spec/unit/coinbase/address/external_address_spec.rb

```diff
@@ -0,0 +1,202 @@
+# frozen_string_literal: true
+
+describe Coinbase::ExternalAddress do
+  let(:network_id) { :ethereum_mainnet }
+  let(:id) { '0x1234' }
+  let(:model) { described_class.new(network_id, id) }
```

Comment:
**nit**: `model` has the connotation of "Coinbase::Client::Address" resource in the code/specs.

Can we call this `address`?

Reply:
👍